content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -44,6 +47,11 @@ The Topology manager receives Topology information from the *Hint Providers* as
 The selected hint is stored as part of the Topology Manager. Depending on the policy configured the pod can be accepted or rejected from the node based on the selected hint.
 The hint is then stored in the Topology Manager for use by the *Hint Providers* when making the resource allocation decisions.
 
+### Topology Manager Kubelet Feature Gate
+
+As of Kubernetes version 1.18, the Topology Manager feature gate is enabled by default. 
+For Kubernetes versions 1.16 and 1.17, the Topology Manager feature gate must be set via the Kubelet flag, `--feature-gates=TopologyManager=true`.

```

Comment:
```suggestion
If you are running Kubernetes v1.16 or v1.17, you must run the kubelet with `--feature-gates=TopologyManager=true,...` to enable this feature.
```

Alternative 1 (maybe post release): backport this advice to the v1.16 and v1.17 documentation, and have the documentation in _master_ signpost readers there.
Alternative 2: skip telling readers how to enable it, as the page tells them they need a v1.18 cluster.